index.html

```diff
@@ -8,24 +8,25 @@
     <link rel="stylesheet" href="./src/styles.css">
 </head>
 <body>
-    <H1>Hi I'm Eli, I like <a href="#pets">my pets</a> and <a href="#kayak">kayaking!</a> Below are my pets and a couple fun  words about kayaking.</H1>
+    <h1>Hi I'm Eli, I like <a href="#pets">my pets</a> and <a href="#kayak">kayaking!</a> Below are my pets and a couple fun  words about kayaking.</h1>
     <h2 id="pets">My Pets are and look like:</h2>
     <ul>
         <li><a href="#smoke">Smokey Joe |</a></li>
-        <li><a href="#rory">Rory |</a></li>
+        <li><a href="#rorz">Rory |</a></li>
         <li><a href="#tyb">Tybalt |</a></li>
-        <li><a href="#ranger">Ranger</a></li>
+        <li><a href="#range">Ranger |</a></li>
+        <li><a href="#quiz">Take a Quiz!</a></li>
     </ul>
     <h3 id="smoke">Smokey Joe:</h3>
     <p>Obsessing over baby chicks!"cuckadoodledoo!"</p>
     <img id="myPets" src="./Assests/Smokey Joe.jpg" alt="Picture of Pyrenese mixbreed dog">
```

Comment:
The id attribute should always be unique. If you want to use the myPets selector on multiple img tags then use a class. `class="myPets"`